docs/user/user.yaml

```diff
@@ -182,3 +182,61 @@ paths:
                 status: 404
                 code: 'USER_NOT_FOUND'
                 message: 'User with the specified email or id was not found.'
+    patch:
+      security:
+        - bearerAuth: []
+      tags:
+        - Users
+      summary: Find and update user by ID
+      description: Finds and updates a user with the specified ID.
+      produces:
+        - application/json
+      parameters:
+        - name: id
+          in: path
+          required: true
+          description: ID of the user that needs to be updated
+          type: string
+      requestBody:
+        required: true
+        description: Provide required data to update user.
+        content:
+          application/json:
+            schema:
+              $ref: '#/definitions/user'
+            example:
+              nativeLanguage: Ukrainian
+              photo: very-cute-cat-photo.jpg
+      responses:
+        204:
+          description: No Content
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/Error'
+              example:
+                status: 400
+                code: 'INVALID_ID'
+                message: 'ID is invalid.'
+        401:
```

Comment:
it must have 403 error as well, aka forbidden

Reply:
added

Reply:
and also pls check allowed fields for admin updating